met-api/src/met_api/services/project_service.py

```diff
@@ -28,11 +28,7 @@ def update_project_info(eng_id: str) -> EngagementModel:
             engagement_metadata: EngagementMetadataModel
             engagement, engagement_metadata = ProjectService._get_engagement_and_metadata(eng_id)
 
-            if not (project_id := engagement_metadata.project_id):
-                # EPIC is not interested in the data without project Id.So Skip the process.
```

Comment:
Will the process be skipped every time because project_id is never available? If so, should more code be removed?

Reply:
I think most likely yes and yes to your question. The ticket didn't give explicit direction for how to handle "Projects" so I decided to leave this for now until we get a "Remove project functionality from MET" ticket